decide/src/main/java/org/example/Decide.java

```diff
@@ -111,7 +111,12 @@ public boolean condition12() {
     }
 
     public boolean condition13() {
-        return true;
+        return condition8()
+                && IntStream.range(0, settings.NUMPOINTS - settings.PARAMETERS.A_PTS - settings.PARAMETERS.B_PTS - 2).anyMatch(
+                (index) -> Point.smallestCircleRadius(settings.POINTS[index],
+                        settings.POINTS[index + settings.PARAMETERS.A_PTS + 1],
+                        settings.POINTS[index + settings.PARAMETERS.A_PTS + settings.PARAMETERS.B_PTS + 2])
+                        < settings.PARAMETERS.RADIUS2);
```

Comment:
I think it should be `<= settings.PARAMETERS.RADIUS2` since we allow points to be on the line too